pages/events/index.vue

```diff
@@ -10,11 +10,37 @@
           </cv-tabs>
         </client-only>
       </div>
+      <div class="event-page__filters-region">
+        <client-only>
+          <cv-multi-select
+            class="event-age__filters-region_dropdown"
+            :theme="'light'"
+            :label="'All locations'"
+            :inline="false"
+            :options="getOptions('regions')"
+            :selection-feedback="'fixed'"
+            @change="options => updateFilters('regionFilters', options)"
+          />
+        </client-only>
+      </div>
+      <div class="event-page__filters-type">
+        <client-only>
+          <cv-multi-select
```

Comment:
Please replace the string properties the same way that I mention for the previous multi-select